opm/autodiff/StandardWellsDense.hpp

```diff
@@ -1283,6 +1297,10 @@ enum WellVariablePositions {
 
                             break;
                         }
+
```

Comment:
I wanted to add one line to separate these two parts. I will reduce the 4 blank lines to 1 line.